rega-genotype/src/main/java/rega/genotype/BlastAnalysis.java

```diff
@@ -400,8 +400,9 @@ private Result compute(SequenceAlignment analysisDb, AbstractSequence sequence)
                 cmd = blastPath + blastCommand + " " + blastOptions
                 	+ " -i " + query.getAbsolutePath()
                     + " -m 8 -d " + db.getAbsolutePath();
+                
```

Comment:
Please remove all the changes to the whitespaces that have no effect on this file